code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm

```diff
@@ -132,8 +132,8 @@
 		consume_bait()
 
 /mob/living/simple_animal/hostile/asteroid/basilisk/watcher/proc/consume_bait()
-	var/obj/item/stack/ore/diamond/diamonds = locate(/obj/item/stack/ore/diamond) in oview(src, 9)
-	var/obj/item/pen/survival/bait = locate(/obj/item/pen/survival) in oview(src, 9)
+	var/obj/item/stack/ore/diamond/diamonds = locate(/obj/item/stack/ore/diamond) in oview(src, 1)
+	var/obj/item/pen/survival/bait = locate(/obj/item/pen/survival) in oview(src, 1)
```

Comment:
You could also remove the distance checks from the two since if they are found they are already at the right distance

Reply:
you're right but this is a webedit PR cause I did it while pissed so gimme a sec and do it properly